exspy/signals/eels.py

```diff
@@ -25,16 +25,14 @@
 from scipy import constants
 from prettytable import PrettyTable
 
+import hyperspy
```

Comment:
Why import the whole `hyperspy` - I only see lines calling the api imported in the next line